plugins/uploadimage/plugin.js

```diff
@@ -88,6 +88,9 @@
 					this.replaceWith( '<img src="' + upload.url + '" ' +
 						'width="' + width + '" ' +
 						'height="' + height + '">' );
+					
+					// Fire change event in order to get updated content.
```

Comment:
Let's add an issue reference.
```suggestion
  // Fire change event in order to get updated content(#1180).
```